compiler/rustc_parse/src/lexer/mod.rs

```diff
@@ -131,7 +131,7 @@ impl<'a> StringReader<'a> {
     ) -> DiagnosticBuilder<'a, !> {
         self.sess
             .span_diagnostic
-            .struct_span_fatal(self.mk_sp(from_pos, to_pos), &format!("{}: {}", m, escaped_char(c)))
+            .struct_span_fatal(self.mk_sp(from_pos, to_pos), &format!("{m}: {}", escaped_char(c)))
```

Comment:
Mixing different types of capture in one format string isn't nice.

Reply:
I think it should be fine, since they are all pretty short and in order. But, I can revert those if necessary.